grr/server/grr_response_server/gui/ui/components/expandable_hash/expandable_hash.ts

```diff
@@ -1,10 +1,26 @@
-import { Component, Input, ChangeDetectorRef } from '@angular/core';
+import { Component, Input } from '@angular/core';
 import { Hash } from '../../lib/api/api_interfaces'
 
-const MAX_TIME_MOUSE_IN_FLIGHT_MS = 200;
+/** Functionality to aggregate a number of (hashType, hashValue) pairs into a single string */
+export class HashTextAggregator {
```

Comment:
(I don't feel too strongly about removing `HashTextAggregator` class, so feel free to leave it and take this comment as educational.) 

This is somewhat verbose code that is only used in one place, where it could be inlined. The logic behind concatenating these strings is neither complex enough to warrant abstraction, nor reused in other code places to warrant extraction.